### PR TITLE
Ah yes, I love democracy.

### PR DESCRIPTION
**Pull request by vini2003**

### CODE REVIEW

File: src/main/java/com/fabriccommunity/spookytime/enchantment/BeheadingEnchantment.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import com.fabriccommunity.spookytime.registry.SpookyEnchantments;
 import com.fabriccommunity.spookytime.registry.SpookyItems;
 import com.google.common.collect.ImmutableList;
+import net.minecraft.block.AnvilBlock;
```
Comment (Juuxel) [inline src/main/java/com/fabriccommunity/spookytime/enchantment/BeheadingEnchantment.java]: Is this used somewhere?

File: src/main/java/com/fabriccommunity/spookytime/mixin/LivingEntityMixin.java [hunk 2]
```diff
@@ -23,29 +23,29 @@
...
+						livingEntity.dropStack(new ItemStack(Items.WITHER_SKELETON_SKULL));
+					} else if (livingEntity.getType() == EntityType.SKELETON) {
+						livingEntity.dropStack(new ItemStack(Items.SKELETON_SKULL));
+					} else if (livingEntity.getType() == EntityType.ZOMBIE) {
+						livingEntity.dropStack(new ItemStack(Items.ZOMBIE_HEAD));
+					} else if (livingEntity.getType() == EntityType.CREEPER) {
+						livingEntity.dropStack(new ItemStack(Items.CREEPER_HEAD));
+					} else if (livingEntity.getType() == EntityType.PLAYER) {
+						livingEntity.dropStack(new ItemStack(Items.PLAYER_HEAD));
+					}
```
Comment (Juuxel) [inline src/main/java/com/fabriccommunity/spookytime/mixin/LivingEntityMixin.java]: Double indentation here?

File: src/main/java/com/fabriccommunity/spookytime/enchantment/BeheadingEnchantment.java (hunk 1 above)
Comment (vini2003) [reply src/main/java/com/fabriccommunity/spookytime/enchantment/BeheadingEnchantment.java]: Oh, yeah, no. Still not used to IDEA's import organization so I missed it.

File: src/main/java/com/fabriccommunity/spookytime/mixin/LivingEntityMixin.java (hunk 2 above)
Comment (vini2003) [reply src/main/java/com/fabriccommunity/spookytime/mixin/LivingEntityMixin.java]: No loner.

**Review by Shrimpstress — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Pull request merged**